index.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>youtube</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="navbar">
+     <div class="header">
+         <a href="#">
+             <span class="icon">
+                <ion-icon name="logo-youtube"></ion-icon>
+             </span>
+             <span class="icon2">
+                 <h2>Youtube</h2>
+             </span>
+             <span class="icon3">
+                 <p>KE</p>
+             </span>
+         </a>
+     </div>
+     <div class="search">
+         <input class="si" type="search" placeholder="Search">
+         <span>
+            <ion-icon class="icons si2" name="search-outline"></ion-icon>
+         </span>
+     </div>
+     <div class="search1">
+        <span class="i1">
+            <ion-icon class="mic icons" id="micro" name="mic-outline"></ion-icon>
+            <div class="text1 mict" id="mict">
+                <p>Create</p>
+             </div>
+        </span>
+        <span class="i1">
+            <ion-icon class="icons" id="create" name="medkit-outline"></ion-icon>
+            <div class="text1">
+                <p>Create</p>
+               </div>
+        </span>
+        <span class="i1">
+            <ion-icon class="icons" name="apps-outline"></ion-icon>
+            <div class="text1">
+                <p>YouTube Apps</p>
+               </div>
+        </span>   
+        <span class="i1">
+            <ion-icon class="icons" name="notifications-outline"></ion-icon>
+            <div class="text1">
+                <p>Notifications</p>
+               </div>
```

Comment:
It will be better if you use Unordered list tags in this code, instead of using span tags

Reply:
It will be better if you use Unordered list tags in this code, instead of using span tags,  this will group the collection of items that do not have a numerical ordering, and their order in the list is meaningless

Reply:
- Great job being polite, phrasing your comment as a suggestion, and giving a reason for the requested change.